src/Exercise5/fibonnacciSequence.js

```diff
@@ -0,0 +1,15 @@
+function fibonacci(num) {
+  let arr = [];
+  let num1 = 0;
+  let num2 = 1;
+  arr.push(num1, num2);
+  let next = num1 + num2;
```

Comment:
instead of initializing those variables num1 and num2 you can initialize them with the array, const arr = [0,1] and then you do the following calculation based on the indexing